otelcollector/scripts/setup.sh

```diff
@@ -64,7 +64,7 @@ cp /etc/cron.daily/logrotate /etc/cron.hourly/
 
 # Install ME
 echo "Installing Metrics Extension..."
-sudo tdnf install -y metricsext2-2.2022.1201.1140
+sudo tdnf install -y metricsext2-2.2023.224.2214-1
```

Comment:
Same here: please update the windows ME too.

Reply:
Sure. Thanks Kaveesh!